tests/dragonfly/snapshot_test.py

```diff
@@ -270,6 +311,13 @@ async def test_snapshot(self, df_seeder_factory, df_server):
         await a_client.connection_pool.disconnect()
 
         assert await seeder.compare(start_capture, port=df_server.port)
+        memory_after = await self._get_info_memory_fields(a_client)
+        for counter, value in memory_before.items():
+            assert memory_after[counter] >= 0.5 * value
```

Comment:
please add a comment why we check > 0.5 *

Reply:
done